inductiva/resources/machines.py

```diff
@@ -63,16 +64,18 @@ def start(self):
             try:
                 logging.info("Creating a machine group. \
                              This may take a few minutes.")
-                start_time = time()
+                start_time = time.time()
+                #TODO: Receive here the machine name and ID.
                 api_instance.create_instance_group(body=instance_group_config)
+                creation_time_mins = (time.time() - start_time) / 60
```

Comment:
is mins reasonable? should it be seconds? I don't have a clue...
Leave it as it is for now. Then we will know better :)

Reply:
I just referred mins to keep with the log above. With more testing we will see what is appropriate.